docs/source/index.rst

```diff
@@ -29,6 +29,10 @@ using Polarsys' Capella_ with Python. Common usage for this API:
 Additionally and as a core idea it provides an interface for the underlying
 database of the Capella model.
 
+Since v0.5, it also supports a simple, but powerful :ref:`declarative modelling
+language <declarative-modelling>`, which is based on the API for the semantic
+model.
+
```

Comment:
I'm feeling weird. Since [there is no option to review unchanged files](https://github.com/community/community/discussions/9099) I add my code suggestion for `conf.py` in this comment.

```
extensions = [
    "sphinx.ext.autodoc",
    "sphinx.ext.coverage",
    "sphinx.ext.extlinks",
    "sphinx_rtd_theme",
    "jinja_in_rst",
    "capellambse.sphinx",
    "sphinx.ext.autosectionlabel",
]
# ...
# -- Options for extlinks ----------------------------------------------------
extlinks = {
    "version": (
        "https://github.com/DSD-DBS/py-capellambse/releases/tag/%s",
        "%s",
    )
}
```

Reply:
Just ignore this for once. In future PRs I'll stash these and spawn a completely new PR after the merge.